src/mainwindow.cpp

```diff
@@ -3062,7 +3062,7 @@ void MainWindow::ignoreMissingData_clicked()
       int row_idx = idx.data(Qt::UserRole + 1).toInt();
       ModInfo::Ptr info = ModInfo::getByIndex(row_idx);
       //QDir(info->absolutePath()).mkdir("textures");
-      info->testValid();
+      info->diskContentModified();
```

Comment:
I think that info->testValid(); here was no longer necessary. Before mo2 would create an empty texture folder inside the mod if the user wanted to mark it as valid. I changed it so that it would instead mark it in the ini, so no actual filesystem change is happening here, so testValid() was originally used to update the status after the new folder was added, while now that is not necessary as there is no filesystem change. Some testing would be good to verify that there isn't some issue with the ui not updating or something when removing that call. The commented out QDir line above should also be cleaned up. That was my bad.